CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ option(FULLY_STATIC
        "Produce a fully statically linked qTox binary (experimental)" OFF)
 option(GUI_TESTS
        "Enable GUI rendering tests, likely tied to a specific Qt version" OFF)
+option(
+  REPR_RCC
```

Comment:
Can you set this flag in the debug builds? We don't build release binaries out of those, so just to make sure this keeps working and doesn't bit-rot, we should exercise it: add something [here](https://github.com/TokTok/qTox/blob/master/.ci-scripts/build-qtox-linux.sh#L77) like this:

```sh
if [ "$BUILD_TYPE" = "Debug" ]; then
  CMAKE_ARGS+=("-DREPR_RCC=ON")
fi
```